1.0.0 Changes

cs-abhishek-shukla opened the pull request:
- Re-wrote Readme
- Rewrote Usage
- Edited setup for type, grammar, and enhanced flow of information

cs-abhinav-goel commented on README.md:
```diff
@@ -0,0 +1,42 @@
+
+- **Publisher**: Fortinet  
+
+- **Compatibility**: 7.3.0 and later
```
Please change the min FSR to 7.4.3 or later

cs-abhinav-goel replied:
@cs-abhishek-shukla

cs-abhishek-shukla replied:
Done

cs-abhinav-goel commented on README.md:
```diff
@@ -0,0 +1,42 @@
+The **Playbook Buttons** widget is a powerful tool designed to streamline your workflow. Here's what it does:
+
+1. **Button Creation**: It allows you to create custom buttons directly on a record's detail view.
+2. **Playbook Execution**: When users click these buttons, they trigger associated playbooks, automating specific tasks.
```
@cs-abhishek-shukla : Not exactly getting the meaning and exact purpose of using this line.

cs-abhishek-shukla replied:
Done

cs-abhinav-goel commented on README.md:
```diff
@@ -0,0 +1,42 @@
+
+1. **Button Creation**: It allows you to create custom buttons directly on a record's detail view.
+2. **Playbook Execution**: When users click these buttons, they trigger associated playbooks, automating specific tasks.
+3. **Context-Aware Display**: The widget intelligently displays relevant playbooks based on the selected record type. For instance, if the record is an *Alert*, you'll see playbooks like:
```
In widget configuration only, it will display all the PBs associated with the particular Module records.

But after adding the widget and saving it, then it will only display the PB which satisfy the visibility condition.

Can we somehow mention this details. rather than the one mentioned by you @cs-abhishek-shukla

cs-abhishek-shukla replied:
Edited

cs-abhinav-goel commented on README.md:
```diff
@@ -0,0 +1,42 @@
+
+4. **Monitoring Progress**: Keep track of playbook execution progress using the *Playbook Execution Wizard*.
+
+In summary, the **Playbook Buttons** widget empowers you to take swift, context-aware actions within FortiSOAR&trade;.
```
Does not understand what exactly is "context-aware actions"

cs-abhishek-shukla replied:
Done

cs-abhinav-goel commented on docs/setup.md:
```diff
@@ -0,0 +1,28 @@
+| Fields                                     | Description                                                                                              |
+|--------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| Select Action Buttons                      | Select playbooks to execute from the record view panel. Select icons for each.                           |
+| Show Playbook Execution Progress On Wizard | Select this option to view selected playbooks' execution on the playbook execution wizard.               |
```
Over here also add a note, that to use this option user should install PB execution wiz. Otherwise this feature will not work.

cs-abhishek-shukla replied:
added

cs-abhinav-goel commented on docs/usage.md:
```diff
@@ -0,0 +1,51 @@
+
+    ![Playbook execution progress on wizard checkbox](./res/edit-view-02.png)
+
+6. Select the playbooks to execute from the **Playbook included list**.
```
Rephrase the Sentence, this field is used to specifically for the PBs to show there execution progress on "PB execution wizard"
Also, the PBs listed here are subset of PB selected in the field "Select Action Buttons".

cs-abhishek-shukla replied:
Done

cs-abhinav-goel commented on docs/usage.md:
```diff
@@ -0,0 +1,51 @@
+|![Widget to display playbook execution](./res/playbook-action-btn-primary.png)|![Widget to display playbook execution](./res/playbook-action-btn-secondary.png)|
+|:--:|:--:|
+|Playbooks as primary buttons|Playbooks as secondary buttons|
+
```
Also, can we add a link to PB execution wiz, widget for the reference.

cs-abhishek-shukla replied:
We must avoid referencing to other repositories as it might break the document.

cs-abhinav-goel reviewed (changes requested):
@cs-abhishek-shukla 
: Please do the changes

cs-abhishek-shukla committed:
1.0.0 Changes
- Added full screen images
- Rewrote setup, readme, and Usage
- Annotated screenshots
- Added release notes
- Added images for playbooks as primary and secondary buttons
- Added info about advanced settings
